target/linux/ramips/image/mt7621.mk

```diff
@@ -104,6 +106,17 @@ define Device/elecom_wrc-1167ghbk2-s
 endef
 TARGET_DEVICES += elecom_wrc-1167ghbk2-s
 
+define Device/elecom_wrc-2533gst
+  DTS := WRC-2533GST
+  IMAGE_SIZE := 11264k
+  DEVICE_TITLE := ELECOM WRC-2533GST
+  ELECOM_GST_FW := 1
```

Comment:
Please create a new/different `elecom-wrc-factory` build command instead.

Maybe you can create a shared command, which is used by the existing and new `elecom-wrc-factory` build command.

Reply:
I see. OK, I'll do it.